modules/changing-the-storage-class-for-an-existing-cluster-storage-instance.adoc

```diff
@@ -0,0 +1,89 @@
+.. Enter a *name* for the cluster storage.
+.. Optional: Enter a *description* for the cluster storage.
+.. Select the needed *storage class* for the cluster storage.
+.. Under *Persistent storage size*, enter a size in gibibytes or mebibytes. 
+.. Under *Connected workbench*, select the workbench with the storage class that you want to change.
+.. Under *Mount folder name*, enter a new storage directory for the cluster storage to mount to. For example, `backup`.
+.. Click *Add storage*.
+
+. Copy the data from the existing cluster storage instance to the new cluster storage instance.
+.. Click the *Workbenches* tab.
+.. Click the action menu (*&#8942;*) beside the workbench with the storage class that you want to change and then click *Start*.
+.. When the workbench status is *Running*, click *Open* to open the workbench.
+.. In JupyterLab, click *File* -> *New* -> *Terminal*.
+.. Copy the data to the new storage directory. Replace _<mount_folder_name>_ with the storage directory of your new cluster storage instance.
++
+[source]
+----
+rsync -avO --exclude='/opt/app-root/src/__<mount_folder_name>__' /opt/app-root/src/ /opt/app-root/src/__<mount_folder_name>__/
+----
++
+For example:
++
+[source]
+----
+rsync -avO --exclude='/opt/app-root/src/backup' /opt/app-root/src/ /opt/app-root/src/backup/
+----
+.. After the data has finished copying, log out of JupyterLab.
+
+. Stop the workbench.
+.. Click the *Workbenches* tab.
+.. Click the action menu (*&#8942;*) beside the applicable workbench and then click *Stop*.
++
+Wait until the *Status* column for the relevant workbench changes from *Running* to *Stopped*.
+
+. Edit the workbench to use the new cluster storage instance.
+.. Click the *Cluster storage* tab.
+.. Click the action menu (*&#8942;*) beside the cluster storage that the workbench currently uses then click *Edit storage*.
+.. Under *Existing connected workbenches*, remove the workbench.
+.. Click *Update*.
+.. Click the *Workbenches* tab.
+.. Click the action menu (*&#8942;*) beside the applicable workbench and then click *Edit workbench*.
+.. In the *Cluster storage* section, under *Use existing persistent storage*, select your new cluster storage.
+.. Click *Update workbench.*
+
+. Restart the workbench.
+.. Click the *Workbenches* tab.
+.. Click the action menu (*&#8942;*) beside the applicable workbench and then click *Start*.
+
+. Optional: The initial cluster storage that uses the previous storage class still appears on the *Cluster storage* tab. If you no longer need this cluster storage (for example, if the storage class is deprecated), you can delete it. 
+
+. Optional: You can delete the mount folder of your new cluster storage instance (for example, the `backup` folder).
+
+.Verification
+* On the *Cluster storage* tab for the project, the new cluster storage instance appears with the needed storage class in the *Storage class* column and the applicable workbench in the *Connected workbenches* column.
```

Comment:
Is the word "needed" necessary here?

Reply:
Maybe there is a better word - I wanted to specify that the storage class was new/different/updated.